README.md

```diff
@@ -231,8 +231,24 @@ To edit your public profile:
     * The background and highlight colors in which your public profile should be displayed.
 
 ### Your storage
-Even if you have navigated to a different Pod, you can still view your storage.
+
+The 'Your storage' menu option brings you to an overview of all the resources part of your Pod. This will always display the storage of your WebID, so your personal storage. Even if you navigate to a different Pod, you can only still view your personal storage.
```

Comment:
```suggestion
The **Your storage** menu item brings you to an overview of all the resources within your Pod. This will always display the storage of your WebID, so it is your personal storage. Even if you navigate to a different Pod, you can still only view your personal storage.
```